src/OidcClient.php

```
@@ -0,0 +1,497 @@
+ * for secure authentication without requiring a client secret.
+ */
+class OidcClient
+{
+    private string $issuer;
+    private string $clientId;
+    private string $redirectUri;
+
+    /**
+     * Cached discovery document.
+     */
+    private ?array $discoveryDoc = null;
```

PHPStan (2.1.33)

Property LiturgicalCalendar\Frontend\OidcClient::$discoveryDoc type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)